src/types/RequiredFieldsTypes.res

```diff
@@ -309,7 +306,7 @@ let rec flattenObject = (obj, addIndicatorForObject) => {
     ->Array.forEach(entry => {
       let (key, value) = entry
 
-      if value->toNullable->RescriptCoreFuture.Nullable.isNullable {
+      if value === JSON.Null {
```

Comment:
@aashu331998 @AashuJuspay please check this condition if its fine

Reply:
not sure lets solve this in next pr